src/components/Home/components/latest_updates/latest_updates.js

```diff
@@ -1,5 +1,93 @@
 import React from "react";
+<<<<<<< HEAD
```

Comment:
This seems to be some leftover code from a rebase. Please remove it and push.

Reply:
okkk